Source/Core/Core/CommonTitles.h

```diff
@@ -10,13 +10,18 @@ namespace Titles
 {
 constexpr u64 BOOT2 = 0x0000000100000001;
 
+constexpr u64 IOS(u32 major_version)
+{
+  return 0x0000000100000000 | major_version;
+}
+
 constexpr u64 SYSTEM_MENU = 0x0000000100000002;
```

Comment:
Should this and the shop constant be moved above the `IOS` function, just to make it more clear that they aren't related? Right now this looks like `IOS(2);` was forgotten here (despite not being an actual IOS value).

Reply:
Didn't think about that. Fixed.